bigquery_etl/dryrun.py

```diff
@@ -69,6 +73,15 @@ def __init__(
         except FileNotFoundError:
             self.metadata = None
 
+        from bigquery_etl.cli.utils import is_authenticated
+
+        if not is_authenticated():
+            print(
+                "Authentication to GCP required. Run `gcloud auth login` "
+                "and check that the project is set correctly."
+            )
+            sys.exit(1)
```

Comment:
Now that authentication is being required in many more cases, it might be good to update the ["GCP CLI tools" section of the README](https://github.com/mozilla/bigquery-etl/blob/b2997d932e179412dcfa786cf4b8a644679bea2b/README.md?plain=1#L20-L23) to instruct non-DEs to authenticate.